src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -130,11 +130,11 @@
 
 (def required-master-files
   "The set of SSL files that are required on the master."
-  [:hostprivkey :hostcert])
+  #{:hostprivkey :hostcert :localcacert})
 
 (def required-ca-files
   "The set of SSL related files that are required on the CA."
-  [:cacert :cacrl :cakey :cert-inventory :serial])
+  #{:cacert :cacrl :cakey :cert-inventory :serial})
```

Comment:
The CA doesn't ever use the `:localcacert` so that would be strange on the surface to have it a required CA file.  Deeper arguments could be made too, but given my previous comment they're moot if we're not going to add `:localcacert` as a required file (to either the master or CA).